### PR TITLE
add ChatDuration support to StartChat example

### PR DESCRIPTION
**Pull request by spencerlepine**
*Issue #, if available:*
👉 Depends on: https://github.com/amazon-connect/amazon-connect-chat-interface/pull/105

*Description of changes:*
Support optional ChatDuration for startChat CFN example. Refer to public documentation: https://docs.aws.amazon.com/connect/latest/APIReference/API_StartChatContact.html#connect-StartChatContact-request-ChatDurationInMinutes

##### Checklist
- [x] update `startChatContactAPI.js` lambda
- [x] Update ChatSDK to [`^2.1065.0`](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md#210650)
- [x] Update startChat `README.md`
- [x] Update `CHANGELOG.md`
- [x] Get PR reviewed
- [x] Deploy latest change to s3 bucket via 2PR
- [x] Test every region deploying a CFN stack
- [x] Create release `1.3.2`

#### S3 Template Updates
```diff
 - startChatContactAPI
   |_ cloudformation.yml
   |_ deployment/
+     |_ ChatSDK.zip
+     |_ start-chat-contact.zip 
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by spencerlepine** [comment]
```diff
 - startChatContactAPI
   |_ cloudformation.yml
   |_ deployment/
      |_ ChatSDK.zip
+     |_ start-chat-contact.zip 
```

File: customChatWidget/public/amazon-connect-chat-interface.js.LICENSE.txt [hunk 1]
```diff
@@ -18,6 +18,13 @@ object-assign
  * @license  MIT
```
Comment (spencerlepine) [inline customChatWidget/public/amazon-connect-chat-interface.js.LICENSE.txt:18]: this is autogenerated

**Comment by spencerlepine** [comment]
## Testing

#### Without `chatDurationInMinutes`

wihtout chagnes

#### With `chatDurationInMinutes`

with duration

File: customChatWidget/README.md [hunk 2]
```diff
@@ -139,6 +139,8 @@ Below steps explain how Custom Chat Widget works on a web page
 					sampleKey1: 'sampleValue1',
 					sampleKey2: 'sampleValue2',
 				},
```
Comment (mhiaror) [inline customChatWidget/README.md:141]: [nit] Indentation!

**Review by mhiaror — APPROVED** [review]

**Comment by spencerlepine** [comment]
## Testing

- [ ] Created new s3 bucket `spenlep-start-chat-contact-duration`
- [ ] Set to public access
- [ ] Uploaded the files via deployment script:
```
+  |_ cloudformation.yml
+  |_ deployment/
+     |_ ChatSDK.zip
+     |_ start-chat-contact.zip 
```
- [ ] Created markdown template deployment button
- [ ] Deployed startChatContactAPI
- [ ] Tested `ChatDurationInMinutes` passed to APIGW in StartChatContact request body
- [ ] Retrieved contaxtId
- [ ] Search ACS/LIMS lambda logs to verify my lambda passed `ChatDurationInMinutes` to Connect

**Review by haomingli2020 — APPROVED** [review]

File: customChatWidget/README.md (hunk 2 above)
Comment (spencerlepine) [reply customChatWidget/README.md:141]: Used spaces not tabs, code editor doesn't pick up. Fixed in next revision

**Review by haomingli2020 — APPROVED** [review]

**Review by mhiaror — APPROVED** [review]

**Pull request merged**